rewrite-maven/src/test/java/org/openrewrite/maven/MavenParserTest.java

```diff
@@ -1251,6 +1262,333 @@ void indirectBomImportedFromParent() {
+                                    <dependencies>
+                                        <dependency>
+                                            <groupId>org.openrewrite.maven</groupId>
+                                            <artifactId>e</artifactId>
+                                            <version>${e.version}</version>
+                                        </dependency>
+                                    </dependencies>
+                                </dependencyManagement>
+                            </profile>
+                        </profiles>
+                    </project>
+                    """
+                )
+              ),
+              mavenProject("a",
+                pomXml(
+                  """
+                    <project>
+                        <parent>
+                            <groupId>org.openrewrite.maven</groupId>
+                            <artifactId>parent</artifactId>
+                            <version>0.1.0-SNAPSHOT</version>
+                            <relativePath />
+                        </parent>
+                        <groupId>org.openrewrite.maven</groupId>
+                        <artifactId>a</artifactId>
+                        <dependencies>
+                            <dependency>
+                                <groupId>org.openrewrite.maven</groupId>
+                                <artifactId>d</artifactId>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.openrewrite.maven</groupId>
+                                <artifactId>e</artifactId>
+                            </dependency>
+                        </dependencies>
+                    </project>
+                    """
+                )
+              ),
+              mavenProject("d",
+                pomXml(
+                  """
+                    <project>
+                        <modelVersion>4.0.0</modelVersion>
+                        <groupId>org.openrewrite.maven</groupId>
+                        <artifactId>d</artifactId>
+                        <version>0.1.0-SNAPSHOT</version>
+                    </project>
+                    """
+                )
+              ),
+              mavenProject("e",
+                pomXml(
+                  """
+                    <project>
+                        <modelVersion>4.0.0</modelVersion>
+                        <groupId>org.openrewrite.maven</groupId>
+                        <artifactId>e</artifactId>
+                        <version>0.2.0-SNAPSHOT</version>
+                    </project>
+                    """
+                )
+              )
+            );
+
+            final AssertionFailedError err = assertThrows(AssertionFailedError.class, fn);
+            assertThat(err.getMessage()).contains("Problem parsing a/pom.xml");  // brittle:(, but class above is broad
+        }
+
+        @Issue("TODO: create issue")
```

Comment:
This test is related to my understanding of [this Maven doc](https://maven.apache.org/settings.html#active-profiles).  It is related, but not the same as what I've written up in #4269, so thinking about creating a new issue for it.  If you think I've misunderstood something or this test is invalid, we can just remove.  This is not a current use-case for us.